src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,22 @@
-const humanCatDogYears = number => {};
+const humanCatDogYears = number => {
+  const humanYears = [number];
+  let catYears = number * 4;
+  let dogYears = number * 5;
+
+  if (number >= 1) {
+    catYears += 11;
+    dogYears += 10;
+  }
+
+  if (number >= 2) {
+    catYears += 5;
+    dogYears += 4;
+  }
+
+  humanYears.push(catYears);
+  humanYears.push(dogYears);
+
+  return humanYears;
+};
```

Comment:
Your code here works perfectly but it took me ages to work out why it worked and if it would work without relying on the tests. It might be good to write some comments to explain why you are doing the calculations in that order. I also didn't expect to see an 11 and 10 when none of the numbers in the instructions used that. I think this is a good example of how your code can be absolutely fine and right, but not necessarily as readable to others. It might make perfect sense for 1 person and confuse another.